docs/guides/joint_fabric_guide.md

```diff
@@ -172,7 +172,7 @@ $ ./jfc-app --rpc-server-port 33055 --storage-directory jfc_b_storage_directory
 -   Commission jf-admin-app
 
 ```
->>> pairing onnetwork 11 110220055 --anchor true
+>>> pairing onnetwork-long 11 110220055 3841 --anchor true
```

Comment:
Considering that jf-admin-apps/lighting-apps are commissioned one by one in the README guide there is no need to specify an additional discriminator parameter. The discriminator would have been helpful if multiple nodes had been advertising in parallel - which we tried to avoid to make the steps as easy as possible. I would suggest trying to avoid an additional parameter if not really needed.

Reply:
both commissioning windows open is not the flow described in the README steps. README describes sequential commissioning. Resolving the conversation as this additional parameter doesn't do any harm (only adds a new parameter for the user to specify).